docs/streams/upgrade-guide.html

```diff
@@ -1634,27 +1652,30 @@ <h3 class="anchor-heading"><a id="streams_api_broker_compat" class="anchor-link"
         <thead>
           <tr>
             <th></th>
-            <th colspan="3">Kafka Broker (columns)</th>
+            <th colspan="4">Kafka Broker (columns)</th>
           </tr>
         </thead>
         <tbody>
           <tr>
             <td>Kafka Streams API (rows)</td>
             <td>0.10.0.x</td>
             <td>0.10.1.x and 0.10.2.x</td>
-            <td>0.11.0.x and<br>1.0.x and<br>1.1.x and<br>2.0.x and<br>2.1.x and<br>2.2.x and<br>2.3.x and<br>2.4.x and<br>2.5.x and<br>2.6.x and<br>2.7.x and<br>2.8.x and<br>3.0.x and<br>3.1.x and<br>3.2.x and<br>3.3.x and<br>3.4.x and<br>3.5.x and<br>3.6.x and<br>3.7.x</td>
+            <td>0.11.0.x and<br>1.0.x and<br>1.1.x and<br>2.0.x</td>
+            <td>2.1.x and<br>2.2.x and<br>2.3.x and<br>2.4.x and<br>2.5.x and<br>2.6.x and<br>2.7.x and<br>2.8.x and<br>3.0.x and<br>3.1.x and<br>3.2.x and<br>3.3.x and<br>3.4.x and<br>3.5.x and<br>3.6.x and<br>3.7.x and<br>3.8.x and<br>3.9.x and<br>4.0.x</td>
           </tr>
           <tr>
             <td>0.10.0.x</td>
             <td>compatible</td>
             <td>compatible</td>
             <td>compatible</td>
+            <td>compatible</td>
```

Comment:
@ijuma just to confirm my reading of KIP-896, the change only affects older clients compatibility with newer brokers, not the other way around, right? In other words a 0.11 client could still run against a 4.0 broker?

Reply:
> In other words a 0.11 client could still run against a 4.0 broker?

I don't think so. According to KIP-896:

> Users relying on clients not included in this document will have to ensure they are using a version that does not rely on protocol API versions that are being removed as part of this proposal. 

This means that broker version 4.x can only communicate with clients version 2.1 and above.

Reply:
Correct @chia7712. This raises the minimum version for everything to 2.1.

Reply:
thanks for confirming -- glad I checked 🙂